src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -266,8 +269,13 @@ private void LoadConfiguration()
             this.MinTxFeeRate = new FeeRate(config.GetOrDefault("mintxfee", this.Network.MinTxFee, this.Logger));
             this.FallbackTxFeeRate = new FeeRate(config.GetOrDefault("fallbackfee", this.Network.FallbackFee, this.Logger));
             this.MinRelayTxFeeRate = new FeeRate(config.GetOrDefault("minrelaytxfee", this.Network.MinRelayTxFee, this.Logger));
+
+            // Node time adjustment
+            this.TimeOffset = this.ConfigReader.GetOrDefault<int>("timeoffset", 0, this.Logger);
```

Comment:
Can you also please adapt the `PrintHelp` and `BuildDefaultConfigurationFile`?
Pleas specify that it is a testing-only launch argument.

Reply:
I think it's better to have that with some explicit warning on what it's for.